contracts/apps/disputable/DisputableAragonApp.sol

```diff
@@ -136,6 +136,7 @@ contract DisputableAragonApp is IDisputable, AragonApp {
     * @param _submitter Address that submitted the action
     * @return Unique identifier for the created action in the context of the Agreement
     */
+   // Note
     function _newAgreementAction(uint256 _disputableActionId, bytes _context, address _submitter) internal returns (uint256) {
```

Comment:
From above, I was thinking about decoupling aragonOS from "agreement"s, and having the concept of disputable action registries.

In that vein, I was thinking of renaming this `_registerDisputableAction()`.